_posts/2016-04-30-akka-basics.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+name: Akka-basics
+title: 'Akka basics'
+date: 2016-04-30 00:20:00 +00:00
+author: Felipe Fernández
+image:
+    src: /assets/img/custom/blog/akka.jpg
+    attribution:
+      text: Akka_20120331 by Helena Whitlock, SGU
+      href: https://www.flickr.com/photos/geologicalsurveyofsweden/8703286181
+tags:
+- akka
+- OOP
+- functional programming
+
+---
+
+As [the previous post](http://codurance.com/2016/04/28/async-systems-with-sync-clients) explained our system is based on Akka. Before going into more detail about our solution, I'd like to explain what is Akka about and why is so awesome.
+
+Akka is a toolkit and not a framework, you can simply use the bits that you need for your service. In this series we'll focus on the core features and we won't go through Akka Stream, Akka-Http or Akka Cluster (since they're out of scope, not because they're not fantastic).
+
+Akka provides a different abstraction to deal with concurrency, paralellism and fault-tolerance. That abstraction is called Actor Model. If you have experience with old Java ecosystem you will know how hard is writing safe and correct multithreaded code. Providing human-friendly abstractions like Actors it's a revolution similar to what Java did with memory management.
```

Comment:
Reading this paragraph I feel you can still say something on what high level problem Akka or Reactive systems solve - maybe even refer to the architectural pattern its related to - Event Driven Architecture (https://en.wikipedia.org/wiki/Event-driven_architecture) if that comes across applicable here.